dist/common_functions.py

```diff
@@ -30,6 +29,40 @@ def last_commit_from_dev():
     return subprocess.run(f"git rev-parse {commit_on_dev}",
         shell=True, capture_output=True, text=True).stdout.strip()
 
+
+def is_fast_mode(fast = None):
+    fast = fast or "-F" in sys.argv[1:] or os.environ.get('FAST', '') != ''
+    if fast:
+        os.environ["FAST"] = "1"
```

Comment:
My understanding is that this will change the environment for the shell running this Python script (which is not be true for the shell scripts). I think it'd be worth using something less generic than `FAST`, e.g. `WT_S_CHECK_FAST` (I pulled that out of thin air, feel free to use something else).

Reply:
`os.environ[...]` in python acts exactly like `export {var}` in bash. I.e. it changes the environment variables for the current and all future child processes.
I agree that FAST can be too generic. I don't know of any other well know use of this name, so I consider it to be safe but I'm ok to prefix if with with a WT-specific thing.
Since we don't really use any environment variables, changed it to just `WT_FAST`.

Reply:
Thanks!